client/src/components-page/profile/profile.module.scss

```diff
@@ -0,0 +1,126 @@
+@import '../../styles/global.styles';
+
+// PROFILE
+.profile {
+
+  // PROFILE CONTAINER
+  .profileContainer {
+   
+    // HEADER SECTION
+    .profileHeader {
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      justify-content: space-between;
+      padding: 1rem;
+      margin-bottom: 1rem;
+
+      background: lighten($lightgrey, 7.5%);
+
+      .name {
+        margin-bottom: 0;
+      }
+
+      .friends {
+        margin-top: 1rem;
+      }
+    }
+
+    // CONTENT SECTION
+    .profileContent {
+      display: flex;
+      flex-direction: column;
+      padding: 0 1rem;
+
+      .avatar {
+        height: 100%;
+        margin-right: 1rem;
+      }
+
+      .bioContainer {
+
+        .bioHeader {
+
```

Comment:
I assume this is to be filled out later?

Reply:
Yeah, there's some styling I'll be adding in there not sure what at the moment so it's currently a placeholder